rules/rules.libsonnet

```diff
@@ -14,9 +14,9 @@
             // Reduces cardinality of this timeseries by #cores, which makes it
             // more useable in dashboards.  Also, allows us to do things like
             // quantile_over_time(...) which would otherwise not be possible.
-            record: 'namespace_pod_container:container_cpu_usage_seconds_total:sum_rate',
+            record: 'namespace_node_pod_container:container_cpu_usage_seconds_total:sum_rate',
```

Comment:
We can't rely on the `node` label being present here as that makes an assumption about the scrape-config/relabeling config people use, and we try to do that as little as possible. We either have to not do this, or join a kube-state-metrics metric onto this instead.

Reply:
@liuxu623 was there a resolution to this? `node` isn't a label on `container_memory_working_set_bytes` timeseries.

Reply:
we can add a record rule like
```
{
            record: 'node_namespace_pod_container:container_memory_working_set_bytes',
            expr: |||
              sum by (namespace, pod, container) (
                container_memory_working_set_bytes{%(cadvisorSelector)s, image!="", container!="POD"}
              ) * on (namespace, pod) group_left(node) max by(namespace, pod, node) (kube_pod_info)
            ||| % $._config,
}
```

Reply:
https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/274